cmd/otel-allocator/README.md

```diff
@@ -43,6 +43,12 @@ spec:
           processors: []
           exporters: [logging]
 ```
+## Set up filter_strategy
+`FilterStrategy` determines how to filter targets before allocating them among multiple collectors. `relabel-config` is the currently supported option. With `FilterStrategy`, targets will be consolidated based on Prometheus configuration "relabel_config". By default, filtering is disabled.
```

Comment:
`relabel_config` formatting seems more appropriate.

Shall we as well update the CRD docs https://github.com/open-telemetry/opentelemetry-operator/blob/2e537f04a7bbc35b7c05f61b81ae559da0018204/apis/v1alpha1/opentelemetrycollector_types.go#L203? 


Could the operator automatically set the "correct" value?

Reply:
we could do this by default if we want, but that's probably another issue :)

Reply:
@pavolloffay, thanks a bunch for your feedback. The formatting was fixed and added content to CRD docs. This is my very first PR, let me know if I missed any files/steps. 

To your other comment on setting `relabel-config` as default, I agree with Jacob's comment, we may need get inputs first thus a separate issue is better.